### ISSUE TITLE
New field: short summary

### ISSUE THREAD

**Issue by marisademeglio** [IDPF/epubtestweb#56]
Instead of "notes", accessibility evaluations should have a "short summary" field (publicly displayed on the eval page *and* on the accessibility grid)
